src/main/scala/cromwell/engine/WorkflowDescriptor.scala

```diff
@@ -9,13 +9,12 @@ import ch.qos.logback.core.FileAppender
 import com.typesafe.config.{Config, ConfigFactory}
 import cromwell.binding._
 import cromwell.binding.values.WdlFile
-import cromwell.engine.backend.Backend
+import cromwell.engine.backend.{CromwellBackend, Backend}
 import cromwell.engine.backend.jes.JesBackend
 import cromwell.engine.io.IoManager
 import cromwell.engine.io.gcs.GoogleCloudStorage
 import cromwell.engine.io.shared.SharedFileSystemIoInterface
 import cromwell.engine.workflow.WorkflowOptions
-import cromwell.server.CromwellServer
 import cromwell.util.TryUtil
```

Comment:
That's actually the reason you all catch so many import shenanigans from me. From older versions of intellij I grew a healthy distrust of its auto-import-doo-dad for scala and i often forget to do it by hand